specs/figures/block_data_structures.dot

```diff
@@ -22,12 +22,12 @@ digraph G {
 
         subgraph cluster_header {
             label = "header";
-            struct1 [label = "version | chainID | height | time | lastBlockID | <f5> lastCommitRoot | consensusRoot | stateCommitment | <f8> availableDataRoot | proposerAddress"];
+            struct1 [label = "height | timestamp | lastBlockID | <f3> lastCommitRoot | consensusRoot | stateCommitment | <f6> availableDataRoot | proposerAddress"];
```

Comment:
Not sure it is a good idea to remove the chain ID from the header.

Reply:
The chainID and version is in the consensus parameters. Since those parameters are fixed and unchangeable without a hard fork (I know Tendermint allows changing parameters with approval from validators, but that's something we want to avoid), there's not much point in duplicating all these values in the top-level header when they can instead just be committed to once.

Reply:
Reason is that the chainID in the header is used to add it to the votes. Maybe instead we can hardcode it into our code base instead (and not have it in the header). Not sure. It is necessary in the votes at least. Otherwise a LL validator that uses the same key to sign multiple other chains could e.g. be slashed for that.

Reply:
Just saw your comment. As long as we commit to the value once and it is accessible to sign votes, then that is fine.